pkg/controller/nodeipam/ipam/BUILD

```diff
@@ -35,6 +35,7 @@ go_library(
         "adapter.go",
         "cidr_allocator.go",
         "cloud_cidr_allocator.go",
+        "controller.go",
```

Comment:
ugh...Rebase went awry. I think a bunch of wonky things got into the build now (I was just trying to push a 1 word spelling fix)